sdk/core/azure-core/test/ut/datetime.cpp

```diff
@@ -440,8 +439,7 @@ TEST(DateTime, ParseTimeInvalid2)
       "1971-01-01T00:60:00Z", // minute too big
       "1971-01-01T00:00:70Z", // second too big
       "1971-01-01T00:00:61Z",
-      "1899-01-01T00:00:00Z", // underflow
```

Comment:
Keep this
```suggestion
      "1599-01-01T00:00:00Z", // underflow
```

Reply:
See 1600-01-01 above.
There was duplication in the original test. so this instance is removed completely.

Original implementation was allowing starting 1900, this one starting 1601, so the corresponding underflow date to test would be 1600, not 1599.